module/os/linux/zfs/zpl_file.c

```diff
@@ -550,6 +552,7 @@ zpl_direct_IO(int rw, struct kiocb *kiocb, struct iov_iter *iter, loff_t pos)
 static loff_t
 zpl_llseek(struct file *filp, loff_t offset, int whence)
 {
+if (!(zfs_disable_holey)) {
```

Comment:
To not mess with formatting his could be moved into `if` below.  And lets remove extra parentheses.